code/mob/living/silicon/ai.dm

```diff
@@ -129,6 +129,7 @@ var/global/list/ai_emotions = list("Annoyed" = "ai_annoyed-dol", \
 	var/status_message = null
 	var/mob/living/silicon/deployed_shell = null
 	var/locking = 0
+	HELP_MESSAGE_OVERRIDE(null)
```

Comment:
```suggestion
```
Don't think this is necessary?

Reply:
you need that for the Help verb to appear in the right click menu, it doesn't if you just override `get_help_message`